src/Agent.Listener/CommandArgs.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommandLine;
+
+namespace Microsoft.VisualStudio.Services.Agent.Listener
+{
+    public class CommandArgs
+    {
+        // Commands.
+        public ConfigureAgent Configure { get; set; }
```

Comment:
I think organizing this with the commands together and the nested classes after would be easier to follow. The interleaving of these things is a bit hard to follow for me.

Reply:
That is really great feedback I'll do that!